examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -235,9 +266,25 @@ export default class App extends PureComponent {
     ];
   }
 
+  _renderDebugOptions() {
+    return <DebugPanel onOptionsChange={this._applyDebugOptions}>{this._renderStats()}</DebugPanel>;
+  }
+
   _renderStats() {
+    const {
```

Comment:
I would suggest rendering the stats inside the debug panel. It may require a little bit of work in probe.gl, but I want us to improve the Stats classes as part of this project.